callbacks_features.py

```diff
@@ -204,15 +204,28 @@ def get_breaths(emg: np.array, start_sample: int, stop_sample: int, method: int)
     big_list = np.round_(emg[start_sample:stop_sample], decimals=5)
     slice_length = 100
 
-    index_hold = []
-    for slice in slice_iterator(big_list, slice_length):
-        entropy_index = hf.entropical(slice)
-        index_hold.append(entropy_index)
+    if method == BreathSelectionMethod.ENTROPY.value: 
+        index_hold = []
+        for slice in slice_iterator(big_list, slice_length):
+            entropy_index = hf.entropical(slice)
+            index_hold.append(entropy_index)
 
-    high_decision_cutoff = 0.9 * ((np.max(index_hold)) - (np.min(index_hold))) + np.min(index_hold)
-    decision_cutoff = 0.5 * ((np.max(index_hold)) - (np.min(index_hold))) + np.min(index_hold)
+        high_decision_cutoff = 0.9 * ((np.max(index_hold)) - (np.min(index_hold))) + np.min(index_hold)
+        decision_cutoff = 0.5 * ((np.max(index_hold)) - (np.min(index_hold))) + np.min(index_hold)
 
-    rms_rolled = hf.vect_naive_rolling_rms(index_hold, 100)  # so rms is rms entropy
+        rms_rolled = hf.vect_naive_rolling_rms(index_hold, 100)  # so rms is rms entropy
+
+    elif method == BreathSelectionMethod.VARIABILITY.value:
+        print("done")
+        index_hold = []
+        for slice in slice_iterator(big_list, slice_length):
+            variability_index = hf.variability_maker(slice, segment_size=slice_length, method='variance', fill_method='avg')             ,
+            index_hold.append(variability_index)
+
+        high_decision_cutoff = 0.5 * ((np.max(index_hold)) - (np.min(index_hold))) + np.min(index_hold)
+        decision_cutoff = 0.05 * ((np.max(index_hold)) - (np.min(index_hold))) + np.min(index_hold)

```

Comment:
```suggestion
        variability = hf.variability_maker(slice, segment_size=slice_length, method='variance', fill_method='avg')

        high_decision_cutoff = 0.5 * ((np.max(variability)) - (np.min(variability))) + np.min(variability)
        decision_cutoff = 0.05 * ((np.max(variability)) - (np.min(variabilit ))) + np.min(variability)

```

I tested the current implementation and it was not working. Given the way we are using for computing the variability, it doesn't make sense to go through segments, as `variability_maker` is already using a sliding window. It looks like it is working with the modification I suggested, but please test it to check the correctness of the values. Eventually this needs to be moved to the resurfemg library.